components/dashboard/src/settings/Integrations.tsx

```diff
@@ -266,19 +266,19 @@ function GitProviders() {
                             &nbsp;
                         </div>
                     </div>
-                    <div className="p-0 my-auto flex flex-col w-3/12">
+                    <div className="p-0 my-auto flex flex-col xl:w-3/12 x-4/12">
```

Comment:
**issue**: Columns are a bit squeezed on viewports with `1280px` or smaller. Freeing the permissions column next to this could allow this column to have some more space.